CHANGELOG.md

```diff
@@ -14,6 +14,17 @@ The format is based on [Keep a Changelog].
 > - **Security**: in case of vulnerabilities.
 
 
+## [UNRELEASED]
+
+### Added
+
+- `IBMQJob` now has three new methods: `done()`, `running()`, and
+  `cancelled()`. The methods are used to confirm the current status 
```

Comment:
The wording "used to confirm" sounds really weird. How about "The methods are used to indicate the job status". Also I don't think it's necessary to list the methods individually below as they're pretty self explanatory. These methods are introduced so users don't need to know about `JobStatus`, and saying "job status is foo" kind of defeats the purpose.